game.py

```diff
@@ -20,7 +20,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word and not correct_letter_guess_statuses[user_input]:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+        else:
+            print(f"The letter {user_input} is not in the word.")
+            wrong_guesses_list.append(user_input)
+            wrong_guesses_count += 1
```

Comment:
A common rule of thumb is to perform the an operation and then after it is successful alert the user. This is so that there are no false positives sent to the user just in case of some failure in your code after the alert is since. With the `print` line being placed after it can only execute upon success of the prior lines.